src/components/modals/Modal.vue

```diff
@@ -77,6 +83,32 @@ export default {
       color: red;
     }
   }
+  .maximize-toggle {
+    position: absolute;
+    right: 55px;
+    top: 20px;
```

Comment:
Needs a z-index, otherwise it won't work in many modals.
And while you're at it, why not group it together with the close button inside a container element so that there is only one element that needs the positioning rules?

Reply:
Because I've never wrote in js/vue before - so I tried to minimize how much I changed.